webpack/assets/javascripts/react_app/components/PF4/Bookmarks/BookmarkItems.js

```diff
@@ -14,33 +14,50 @@ import DocumentationUrl from '../DocumentationLink';
 export const addBookmarkItem = ({ canCreate, setModalOpen }) =>
   canCreate && (
     <DropdownGroup key="group 1">
-      <DropdownItem key="newBookmark" id="newBookmark" onClick={setModalOpen}>
+      <DropdownItem
+        ouiaId="new-bookmark-dropdown-item"
+        key="newBookmark"
+        id="newBookmark"
+        onClick={setModalOpen}
+      >
         <PlusIcon /> {__('Bookmark this search')}
       </DropdownItem>
-      <DropdownSeparator key="separator" />
+      <DropdownSeparator
+        ouiaId="new-bookmark-dropdown-separator"
+        key="separator"
+      />
     </DropdownGroup>
   );
 
 const pendingItem = (
-  <DropdownItem key="spinner" className="loader-root" isDisabled>
+  <DropdownItem
+    ouiaId="spinner-dropdown-item"
+    key="spinner"
+    className="loader-root"
+    isDisabled
+  >
     <Spinner size="xs" aria-label="loading bookmarks" />
   </DropdownItem>
 );
 
 const bookmarksList = ({ bookmarks, onBookmarkClick }) =>
   (bookmarks.length > 0 &&
     bookmarks.map(({ name, query }) => (
-      <DropdownItem key={name} onClick={() => onBookmarkClick(query)}>
+      <DropdownItem
+        ouiaId="query-dropdown-item"
```

Comment:
this will create duplicates of the id,
```suggestion
        ouiaId=`{name}-dropdown-item`
```